packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -211,6 +212,23 @@ type HydrationHostConfig<T, P, I, TI, C, CX, PL> = {
   ): void,
 };
 
+// 0 is PROD, 1 is DEV.
+// Might add PROFILE later.
+type BundleType = 0 | 1;
+
+type DevToolsConfig<I, TI> = {|
+  bundleType: BundleType,
+  version: string,
+  rendererPackageName: string,
+  // Note: this actually *does* depend on Fiber internal fields.
+  // Used by "inspect clicked DOM element" in React DevTools.
+  findFiberByHostInstance?: (instance: I | TI) => Fiber,
```

Comment:
I think the devtools integration used to work without this method 🤔 

With this being required I think we’d also need `ReactFiberTreeReflection.findCurrentHostFiber` and `ReactFiberTreeReflection.findCurrentHostFiberWithNoPortals` also exposed.

Reply:
I don't think we need them. This one doesn't have to be the current one. So as long as you save the fiber on some field on the instance (or use a weakmap), you can get one back.

Reply:
It still does. The implementation is:

Reply:
That's also true. (Though it's nice to implement if you can I guess.)